pkg/astnormalization/subgraph_sdl_normalization_test.go

```diff
@@ -9,73 +9,51 @@ import (
 	"github.com/stretchr/testify/assert"
 )
 
-func TestNormalizeSubgraphSDL(t *testing.T) {
-	run := func(name, input, expectedOutput string, norm registerNormalizeFunc) {
-		inDoc, _ := astparser.ParseGraphqlDocumentString(input)
-		outDoc, _ := astparser.ParseGraphqlDocumentString(expectedOutput)
-		expectedOutput, _ = astprinter.PrintString(&outDoc, nil)
-		walker := astvisitor.NewWalker(48)
-		norm(&walker)
-		walker.Walk(&inDoc, nil, nil)
-		input, _ = astprinter.PrintString(&inDoc, nil)
-		t.Run(name, func(t *testing.T) {
-			assert.Equal(t, input, expectedOutput)
-		})
+func runNormalizeSubgraphSDL(t *testing.T, name, input, expectedOutput string, norm ...registerNormalizeFunc) {
```

Comment:
There is already a normalizer run method: https://github.com/TykTechnologies/graphql-go-tools/blob/master/pkg/astnormalization/astnormalization_test.go#L379

Can we reuse it instead of defining a similar function again?

Reply:
the important part in this function is the last argument(`norm ...registerNormalizeFunc`)! we could have them both :)

Reply:
What's wrong with `runMany`? :-)
https://github.com/TykTechnologies/graphql-go-tools/blob/master/pkg/astnormalization/astnormalization_test.go#L408

We should try to avoid adding duplicate code.

Reply:
:) didn't know about this know